### DISCUSSION TITLE
Having Trouble on Laravel 9

### DISCUSSION THREAD

**Discussion by ssgnoe** [beyondcode/laravel-websockets#1072] [Q&A] [answered]
Hey out there, maybe someone can help me.
I migrated from a Lumen 8.40 (on PHP7.4, laravel-websockets 1.12) project to a Laravel 9.0 (on PHP 8.1, laravel-websockets: 1.13). The Websockets generally are working and the presence / online status is connected well in the frontend, but my events from Laravel are not beeing published.
I started debugging dashboard from Lumen and from Laravel Application, both can connect to the running Laravel Websocket Server und the Lumen project can publish testevents. On Laravel I can not publish any events, getting an "ok" from API, but nothing is published.
I have a total copy of config/websockets.php and config/broadcasting.php, as well as Pusher Settings in .env. Is there another file which I missed I should configure correctly? Anything else I missed? Or are there problems on Laravel 9 and PHP 8?

It seems like websocket-server is running well, just the part of Laravel pushing Events to sockets driving me crazy...

Thanks for your help!

**Comment by danh65** [comment, marked as answer]
It sounds like the same problem I was having and found this that solved my problem: https://github.com/beyondcode/laravel-websockets/issues/1041

Basically downgrade pusher server by:
composer require pusher/pusher-php-server:7.0.2

Regards, Dan

**Reply by ssgnoe** [reply]
You're my hero! Hours of searching this error and a that easy solution. Thank you!
